src/keys.test.ts

```diff
@@ -8,4 +9,13 @@ describe('Test for keys', () => {
   it('should return keys of object', () => {
     expect(keys({ a: 'x', b: 'y', c: 'z' })).toEqual(['a', 'b', 'c']);
   });
+
+  describe('strict', () => {
+    const actual = keys.strict({ a: 'x', b: 'y', c: 'z' } as const);
+    expect(actual).toEqual(['a', 'b', 'c']);
+
+    const result: AssertEqual<typeof actual, Array<'a' | 'b' | 'c'>> = true;
```

Comment:
Have never seen this type comparison before. TS would broke here on Jest compile?

Reply:
it's a type level "test". the jest assertion itself doesn't really do anything (compares true with true), but the typescript compiler will complain if the two types are not the same.

Reply:
Look like possible replacement but we'll be required configure eslint for that. Think we can got back to this later, I'll create issue